test/lib/tokenFactory.test.js

```diff
@@ -31,11 +40,21 @@ describe('Token Factory', () => {
             useCleanCache: true,
             warnOnUnregistered: false
         });
+        generateTokenMock = {
+            generateValue: sinon.stub(),
+            hashValue: sinon.stub()
```

Comment:
@FenrirUnbound Should we mock the `hashValue` function here? Making sure the `hashValue` function returns the same hash for the same string is important. And not mocking this should not fail the test. But the crypto is from an external module, I am not sure if we should mock it or not.